codebuild/bin/start_codebuild.sh

```diff
@@ -28,6 +28,7 @@ BUILDS=(
     "s2nUnitNix"
     "Integv2NixBatchBF1FB83F-7tcZOiMDWPH0 us-east-2 batch"
     "kTLS us-west-2 no-batch"
+    "Openssl3fipsWIP us-west-2 no-batch"
```

Comment:
I also noticed that I'd forgotten to turn the Openssl3fipsWIP job on :sweat_smile: It's on now so I have to add it to this list.